### ISSUE TITLE
Save an image located on a HTTP remote server

### ISSUE THREAD

**Issue by elielmathe** [quiply/SaveImage#10]
Can we save a file located on a remote server using this plugin?

**Comment by commonpike** [comment]
Yes (I do that), but to download the file, you need (something like) the File and FileTransfer plugin. After you've downloaded the photo (presumably 'into the app') you can use this plugin to make it appear into the Gallery or Camera Roll.

There are plugins that do both, and they all have quirks and errors.

https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-file-transfer/
